curriculum/examples/sat-prep/math-algebra-sample.json

```
@@ -0,0 +1,582 @@
+          "domain": "algebra",
+          "weight": 0.10,
+          "strategyTags": ["translation", "modeling"]
+        }
+      },
+      "children": [
+        {
+          "id": "sat-algebra-word-translation",
+          "nodeType": "segment",
+          "title": "Translating Words to Equations",
+          "description": "Converting verbal descriptions into mathematical expressions",
+          "content": "Key translations: 'is' = equals, 'of' = multiply, 'per' = divide, 'more than' = add, 'less than' = subtract.",
```

Biome (2.1.2)

[error] The key content was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)